package.json

```
@@ -10,7 +10,7 @@
   },
   "name": "temporal-hello-world",
   "dependencies": {
-    "temporalio": "0.4.2"
+    "temporalio": "0.11.0"
```

```suggestion
    "temporalio": "0.11.x"
```

We now have proper semver